introduction.md

```diff
@@ -0,0 +1 @@
+Hi my name is Tyler Fullington
```

Comment:
echo "On my way to becoming a GitHub Master. I love Ruby and JavaScript." > introduction.md
git commit -am "Added Code School Path"
git push